developer_docs/overview.md

```diff
@@ -56,18 +66,18 @@ Therefore the print functions which are acceptable for visiting this object are:
 
 We run through these possible functions choosing the one which is the most specialised. If none of these methods exist, then an error is raised.
 
-In the case of `NumpyTanh` the function which will be selected is `_print_NumpyUfuncBase` when translating to C or Fortran, and `_print_PyccelInternalFunction` when translating to Python
+In the case of `NumpyTanh` the function which will be selected is `_print_NumpyUfuncBase` when translating to C or Fortran, and `_print_PyccelInternalFunction` when translating to Python.
 
 ### AST
 
 The objects as understood by Pyccel are each described by classes which inherit from [pyccel.ast.basic.Basic](../pyccel/ast/basic.py).
 These classes are found in the [ast](../pyccel/ast) folder.
 The objects in the Abstract Syntax Tree (AST) are described in several files.
-There is one file for each supported extension module and files to group concepts, e.g. literals/operators/built-in functions
+There is one file for each supported extension module and files to group concepts, e.g. literals/operators/built-in functions.
 
 ## Error System
 
-Pyccel tries to fail cleanly and raise readable errors for users. This is managed using the error handling module found in the [errors](../pyccel/errors) folder. In order to raise an error 2 things must be done:
+Pyccel tries to fail cleanly and raise readable errors for users. This is managed using the error handling module found in the [errors](../Pyccel/errors) folder. In order to raise an error 2 things must be done:
```

Comment:
This change is wrong. Pyccel here is the name of a folder that is being linked to. Capitalisation will break the link